src/expressions/if-expr.md

```diff
@@ -1,14 +1,22 @@
-# `if` and `if let` expressions
+# `if` expressions
 
-## `if` expressions
+## Syntax
+
+The same syntax is used by `if`, `if let` and chains of expressions.
```

Comment:
I was perhaps intending maybe a little bit of a greater rewrite here, as I'm not sure I entirely see `if` and `if let` being fundamentally different.  Is there a reason to keep them as separate sections?  

I think what I was thinking is it would say something like:

> The condition operand can be a list of one or more _IfLet_ expressions separated by `&&`.
> If all of the boolean values evaluate to true and all patterns match their scrutinee value, the consequent block is executed and…
> …and include a description of pattern bindings, scopes, `&&` priority, etc.

I guess I'm not seeing why they would be kept separate.  `if EXPR {}` just seems to be a special case where there is an `if` expression with a single boolean value.

Reply:
I don't have enough free time to perform a greater rewrite

Reply:
Do you mind if I push some changes to your branch?

Reply:
Not at all